relay_messages_test.go

```diff
@@ -307,6 +310,68 @@ func TestLazyCallReqSetTTL(t *testing.T) {
 	})
 }
 
+func byteKeyValToMap(tb testing.TB, buffer []byte) map[string]string {
```

Comment:
we already have code very similar to this:
https://sourcegraph.uberinternal.com/github.com/uber/tchannel-go/-/blob/messages.go#L181